package.json

```diff
@@ -59,8 +59,8 @@
     "yargs": "~3.9.1"
   },
   "scripts": {
-    "coverage": "nyc npm run mocha-node-test && nyc report",
-    "coveralls": "npm run mocha-node-test && nyc report --reporter=text-lcov | coveralls",
+    "coverage": "nyc npm run mocha-node-test -- --grep @nycinvalid --invert",
+    "coveralls": "npm run mocha-nyc-test && nyc report --reporter=text-lcov | coveralls",
```

Comment:
@megawac Ouch good catch!  Fixed in https://github.com/caolan/async/pull/1234/commits/00f12da45d6df7b9e5115996d5071ae759531e1e.  Let me know if you want me to rebase.